tests/test_request.py

```diff
@@ -126,11 +126,8 @@ def test_accessing_user_attributes(self):
             assert isinstance(getattr(self.resp.user, letter), str)
 
     def test_headers_are_case_insensitive(self):
-        assert (
-            self.resp.headers["content-type"]
-            == self.resp.headers["CoNtEnT-TyPe"]
-            == "application/json"
-        )
+        assert self.resp.headers["content-type"] == self.resp.headers["CoNtEnT-TyPe"]
```

Comment:
I would write it more direct instead of creating that kind of dependency:
```python
assert self.resp.headers["content-type"] == "application/json"
assert self.resp.headers["CoNtEnT-TyPe"] == "application/json"
```